src/boot/efi/devicetree.c

```diff
@@ -0,0 +1,183 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <efi.h>
+
+#include "devicetree.h"
+#include "util.h"
+
+#define FDT_V1_SIZE (7*4)
+
+#ifndef DEVICE_TREE_GUID
+#define DEVICE_TREE_GUID \
```

Comment:
gnu-efi provides this as EFI_DTB_TABLE_GUID, and then you can move this to missing_efi.h